lib/init.g

```diff
@@ -705,6 +705,21 @@ DeclareUserPreference( rec(
   check:= val -> IsInt( val ) and 0 <= val,
   ) );
 
+DeclareUserPreference( rec(
+  name := "Autocompleter",
+  description := [
+                   "Set how names are filtered during tab-autocomplete, \
+this can be: \"default\": case-sensitive matching. \"case-insensitive\": \
+case-insensitive matching, or a record of two functions, each of which take \
+two arguments. 'filter' takes a list of names and a partial identifier and returns \
```

Comment:
Maybe clarify what `filter` is, namely the name of a record component? So:
`or a record with two components named 'filter' and 'completer' which both must be set to functions: 'filter' takes...`